.travis.yml

```diff
@@ -14,11 +14,11 @@ env:
     - SUITE=pg
     - SUITE=sqlite3
 rvm:
-  - 2.2.3
```

Comment:
I'd keep both and also add 2.4 to the list, we want to make sure all ruby versions works unless we have a strong reason to remove support to 2.2.